### ISSUE TITLE
[TI-81] Allow correction with filtered ids

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/timewarrior#85]
_Plup on 2017-06-16T19:42:30Z says:_ 

 It would great to have a way to modifiy tags with (at)ids.

If I look for all recorded tasks with a tag filter I get this :
```
$ timew sum :year :ids vhenry
Wk Date Day ID Tags Start End Time Total
W20 2017-05-19 Fri @15 régler le problème de mails en spam, vhenry, work 6:43:31 9:16:26 2:32:55 2:32:55
W22 2017-05-29 Mon @14 migrer le site, vhenry, work 15:00:00 17:11:49 2:11:49 2:11:49
W22 2017-05-30 Tue @13 migrer le site, vhenry, work 10:36:57 10:54:58 0:18:01 0:18:01
W22 2017-06-01 Thu @12 migrer le site, vhenry, work 19:10:44 22:08:46 2:58:02 2:58:02
W22 2017-06-02 Fri @11 migrer le site, vhenry, work 16:20:00 16:59:32 0:39:32 0:39:32
W23 2017-06-05 Mon @10 migrer le site, vhenry, work 11:38:46 12:00:35 0:21:49 0:21:49
W23 2017-06-06 Tue @9 migrer le site, vhenry, work 10:01:00 10:12:46 0:11:46
@8 migrer le site, vhenry, work 12:48:27 13:29:42 0:41:15
@7 migrer le site, vhenry, work 19:15:09 20:10:16 0:55:07 1:48:0```
Now if I want to add/remove a tag or make a correction I could do this :
```
$ timew tag @8 something```
But in this context @8 is not the same record anymore. So I'm actually adding a tag to the 8th last record in the database.

Is there a workaround ?

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-06-16T19:42:30Z
Modified: 2018-01-12T06:22:18Z
```

**Comment by taskwarrior** [comment]
_Plup on 2017-06-16T19:45:28Z says:_ 

 Oops. Is there a way to edit the message to add the right code tags ?

**Comment by taskwarrior** [comment]
_Plup on 2017-06-16T19:55:44Z says:_ 

 As a workaround I remove the "tag" filter and pipe the command in grep "tag". This gives me the write ID.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-06-16T22:27:12Z says:_ 

 Code tags are \{ n o f o r m a t } without the spaces.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-06-16T22:28:28Z says:_ 

 No workaround at the moment - you have to show an unfiltered list to use `@`.  Good idea.

**Comment by taskwarrior** [comment]
_Thomas Lauf on 2018-01-12T06:20:36Z says:_ 

 This got fixed with commit 182cd5b0b4f81688c99852c721de5645a03207cb
